packages/server/src/api/muster/muster.postgres.controller.ts

```diff
@@ -0,0 +1,53 @@
+import { Response } from 'express';
+import moment from 'moment-timezone';
+import { GetMusterRosterQuery, Paginated } from '@covid19-reports/shared';
+import { assertRequestQuery } from '../../util/api-utils';
+import { ApiRequest, OrgRoleParams} from '../api.router';
+import { getObservations } from '../observation/observation.model';
+import { Log } from '../../util/log';
+import { toMusterCompliance, MusterCompliance} from './muster.compliance';
+import { getEdipi, getRosters, getUnitIds } from '../../util/roster-utils';
+import { getMusterTimeView } from './muster.timeview';
+
+
+class MusterPostgresCtr {
+
+  /**
+   * Provides Muster Compliance details for all service members for the given unit id(s)
+   */
+  async getMusterRoster(req: ApiRequest<OrgRoleParams, null, GetMusterRosterQuery>, res: Response<Paginated<Partial<MusterCompliance>>>) {
```

Comment:
This suggestion is wordy but perhaps a more appropriate name is `getUserMusterComplianceForRosters` or something?

Reply:
renamed